WASM Build Error

Someone on the Slack channel pointed out he is getting a build error when compiling to WASM.

```
Compiling meilisearch-sdk v0.7.0 (X:\Work\meilisearch-rust-0.7.0)
error[E0599]: no method named `try_into` found for type `u128` in the current scope
   --> src\progress.rs:152:38
    |
152 |                 interval.as_millis().try_into().unwrap(),
    |                                      ^^^^^^^^ method not found in `u128`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use std::convert::TryInto;`
error: aborting due to previous error
For more information about this error, try `rustc --explain E0599`.
error: could not compile `meilisearch-sdk`
To learn more, run the command again with --verbose.
Error: Compiling your crate to WebAssembly failed
Caused by: failed to execute `cargo build`: exited with exit code: 101
  full command: "cargo" "build" "--lib" "--release" "--target" "wasm32-unknown-unknown"
```

The error happens [here](https://github.com/meilisearch/meilisearch-rust/blob/39aeac1835dffeefac9718d58d33d842b7be1169/src/progress.rs#L152). When I compile with the test compile from the docs, `cargo build --example web_app`, I don't get an error, but if I compile like he compiled, `cargo build --lib --release --target wasm32-unknown-unknown` I get the same error.

I tested changing the interval line to `interval.as_millis()` I get an error to change it to `interval.as_millis().try_into().unwrap()` which is what is already there.

```
rror[E0308]: mismatched types
   --> src/progress.rs:152:17
    |
152 |                 interval.as_millis(),
    |                 ^^^^^^^^^^^^^^^^^^^^ expected `i32`, found `u128`
    |
help: you can convert a `u128` to an `i32` and panic if the converted value doesn't fit
    |
152 |                 interval.as_millis().try_into().unwrap(),
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error
```

--- a/src/progress.rs
+++ b/src/progress.rs
@@ -27,7 +27,6 @@ pub struct Progress<'a> {
 }
 
 impl<'a> Progress<'a> {
-    ///
     /// # Example
     ///
     /// ```
@@ -143,6 +142,7 @@ pub(crate) async fn async_sleep(interval: Duration) {
 #[cfg(target_arch = "wasm32")]
 pub(crate) async fn async_sleep(interval: Duration) {
     use wasm_bindgen_futures::JsFuture;
+    use std::convert::TryInto;
 
     JsFuture::from(js_sys::Promise::new(&mut |yes, _| {
         web_sys::window()

